examples/test_certs/pq-osa-ca.crt

```diff
@@ -1,29 +1,29 @@
 -----BEGIN CERTIFICATE-----
-MIIFCzCCAvOgAwIBAgIUavME2jj8LFyR4tz95Uc8eKKrYmswDQYJKoZIhvcNAQEL
```

Comment:
How does this change relate to the addition of cargo deny?

Reply:
Not directly. 
But when i tested the example when updating the `liboqs`to 0.8.0, found CA cert is outdated and example run is broken.
Included it since its a minor change.

Please let me know if it is better to create a separate pr for this

Reply:
No need for a separate PR but a separate commit (with an appropriate commit message) would be good.